### PR TITLE
Sync `v2-2-stable` with `v2-2-test` to release `2.2.5`

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

**Comment by potiuk** [comment]
wowowow!

**Comment by potiuk** [comment]
Hey @ephraimbuddy  - I thiink we should merge https://github.com/apache/airflow/pull/22385 before we run 2.2.5 - we have some problems with the currently "latest" providers and we need that one and at least voting on the new providers to release 2.2.5.

**Comment by potiuk** [comment]
I'd really love to merge it and start new provider's wave.

File: airflow/config_templates/config.yml [hunk 1]
```diff
@@ -1767,6 +1767,14 @@
       type: string
       example: ~
       default: "30"
+    - name: deactivate_stale_dags_interval
+      description: |
+        How often (in seconds) to check for stale DAGs (DAGs which are no longer present in
+        the expected files) which should be deactivated.
+      version_added: 2.3.0
```
Comment (eladkal) [inline airflow/config_templates/config.yml]:
```suggestion
      version_added: 2.2.5
```

BTW is it a feature or a bug fix?
Maybe the PR that added it should not be targeted to 2.2.5?

Comment (ephraimbuddy) [reply airflow/config_templates/config.yml]: It's a bug fix. it was previously targeted for 2.3.0. Thanks for this

**Review by potiuk — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Looks Great

**Comment by potiuk** [comment]
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Pull request merged**